examples/decoding/plot_haxby_different_estimators.py

```diff
@@ -95,8 +95,8 @@
     print(70 * '_')
 
     for category in categories:
-        classification_target = stimuli[
-            np.logical_not(resting_state)] == category
```

Comment:
You don't need parentheses here

```python
classification_target = stimuli[task_mask] == category
```

but it could be argued that it makes it slightly more readable (not entirely convinced personally). @AlexandreAbraham what do you think?